### PR TITLE
roberta base openai detector

### PR DESCRIPTION
**Pull request by 1Maxnet1**
I implemented a first evaluator

### CODE REVIEW

File: src/fool_ai_detector/service/roberta_base_openai_detector.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+"""
+Evaluator based on the roberta_base_openai_detector
```
Comment (EhrenbruderRoschtbraten) [inline src/fool_ai_detector/service/roberta_base_openai_detector.py:2]:
```suggestion
Evaluator based on the roberta_base_openai_evaluator
```
We call it evaluator.

File: src/fool_ai_detector/service/roberta_base_openai_detector.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+class RobertaBaseEvaluator(Evaluator):
+    """
+    Base class for an Evaluator
+    """
+
+    def evaluate(self, input_file_path) -> bool:
+        """
+        Evaluates a given text based on the specific detector
```
Comment (EhrenbruderRoschtbraten) [inline src/fool_ai_detector/service/roberta_base_openai_detector.py:15]:
```suggestion
        Evaluates a given text based on the specific evaluator
```

File: test/fool_ai_detector/service/test_roberta_base_evaluator.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+from fool_ai_detector.service.roberta_base_openai_detector import RobertaBaseEvaluator
```
Comment (EhrenbruderRoschtbraten) [inline test/fool_ai_detector/service/test_roberta_base_evaluator.py:1]:
```suggestion
from fool_ai_detector.service.roberta_base_openai_evaluator import RobertaBaseEvaluator
```

**Review by EhrenbruderRoschtbraten — COMMENTED** [review]
Only change detector to evaluator

File: src/fool_ai_detector/service/roberta_base_openai_detector.py (hunk 1 above)
Comment (EhrenbruderRoschtbraten) [reply src/fool_ai_detector/service/roberta_base_openai_detector.py]: Only model name, will resolve this.

File: src/fool_ai_detector/service/roberta_base_openai_evaluator.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+class RobertaBaseEvaluator(Evaluator):
+    """
+    Base class for an Evaluator
+    """
+
+    def evaluate(self, input_file_path) -> bool:
+        """
+        Evaluates a given text based on the specific detector
+        :param input_file_path: path to the file, that should be evaluated
+        """
```
Comment (LeviUniAcc) [inline src/fool_ai_detector/service/roberta_base_openai_evaluator.py:17]:
```suggestion
        :return: bool; true = fake, false = real
        """
```

**Review by LeviUniAcc — APPROVED** [review]
maybe add return comment, up to you. LGTM

**Review by EhrenbruderRoschtbraten — APPROVED** [review]

**Pull request merged**